.gitignore

```diff
@@ -35,6 +35,7 @@ _site
 packages
 .packages
 .dart_tool
+*.g.dart
```

Comment:
Can we remove this? I would think that any `.g.dart` files that are created outside of `.dart_tool` should be tracked.

Reply:
+1